draft-ietf-quic-transport.md

```diff
@@ -290,9 +290,38 @@ across changes in the client's and the server's network addresses.
 # Packet Types and Formats
 
 We first describe QUIC's packet types and their formats, since some are
-referenced in subsequent mechanisms.  Note that unless otherwise noted, all
-values specified in this document are in little-endian format and all field
-sizes are in bits.
+referenced in subsequent mechanisms.
+
+All values specified in this document are in big-endian format and all field
+sizes are in bits.  When discussing individual bits of fields, the least
+significant bit is referred to as bit 0.  Hexadecimal notation is used for
+describing the value of fields.
+
+
+## Versions
+
+QUIC versions are identified using a 32-bit value.
+
+The version 0x00000000 is reserved to represent an invalid version.  This
+version of the specification is identified by the number 0x00000001.
+
+Versions with the most significant 16 bits of the version number cleared are
+reserved for use in future IETF consensus documents.
+
+\[\[RFC editor: please remove the remainder of this section before
+publication.]]
+
+The version number for the final version of this specification (0x00000001), is
+reserved for the version of the protocol that is published as an RFC.
+
+Version numbers used to identify IETF drafts are created by adding the draft
+number to 0xff000000.  For example, draft-ietf-quic-transport-13 would be
+identified as 0xff00000D.
+
+Versions of QUIC that are used for experimentation are coordinated on the
```

Comment:
Should we reserve a range for experimentation as well as suggesting where to coordinate?